Client/Cards/index.ts

```diff
@@ -111,7 +111,15 @@ export class Cards extends List<model.CardResponseV2 | model.CardResponse> {
 		)
 		return this.extractResponse<model.CardTypeResponseV2>(response)
 	}
-	//Possibly should be moved to its own class
+
+	async setCardTypesV2(cardTypeProfileRequest: model.CardTypeProfileRequest) {
+		const response = await this.connection.post<model.CardTypeProfileResponse>(
+			"v2/card/types/profiles",
```

Comment:
this path + POST creates a new profile, and the name of the method leads me to believe it's not what was intended.

Reply:
Changed name to createCardTypesV2